src/components/RoomCalendar.tsx

```
@@ -126,6 +143,39 @@ export default function RoomCalendar({ roomNumber, onClose }: RoomCalendarProps)
     }
   }
 
+  const handleEventClick = (clickInfo: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)